.gitignore

```diff
@@ -7,3 +7,4 @@ data/exports/*
 .clever.json
 .idea/
 dist
+package-lock.json
```

Comment:
@dienamo Il faut le laisser en commit 
https://dev.to/adamklein/package-lock-json-in-git-or-not-50l5